scheduler/filter/README.md

```diff
@@ -79,6 +80,73 @@ Those tags are sourced from `docker info` and currently include:
 * kernelversion
 * operatingsystem
 
+## Affinity Filter
+
+#### Containers
+
+You can schedule 2 containers and make the container #2 next to the container #1.
+
+```
+$ docker run -d -p 80:80 --name front nginx
+ 87c4376856a8
+
+$ docker ps
+CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS                           NODE        NAMES
+87c4376856a8        nginx:latest        "nginx"             Less than a second ago   running             192.168.0.42:80->80/tcp         node-1      front
+```
+
+Using `-e affinity:container=front` will schedule a container next to the container `front`.
```

Comment:
Note for later: We should update the example using regular expressions as it makes more sense for affinity.

e.g.

`--name front_1`
`--name front_2`
`-e affinity:container=front_*`